### ISSUE TITLE
Add ability to autoplay episodes from the same podcast after the end of a podcast is reached

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#554]
> For example, I subscribe to a new podcast, and then I want to start playing from some particular episode, then through the episodes sequentially from that point without downloading them. I don’t want to have to manually add them to a queue, because, for example, let’s say that after I’ve listened to 2 or 3 episodes of that first podcast, say I want to jump to another podcast and start playing from some particular episode, then through episodes sequentially from that point. If I added the episodes from the first podcast to a queue, those episodes previously added to the queue would play first, or I would have to manually delete them in order to play the second podcast as I describe.

> With the Apple podcast app, I can start with a particular episode of a podcast, and start playing them sequentially without having to download them or add them to a queue, then—when I want to switch to another podcast—I simply go to that second podcast, start play with the episode I choose, and from there, the episodes of this second podcast will play sequentially until I stop. I do not have to manually create a queue, or do not have to download episodes in order to play them sequentially.

**Comment by mitchdowney** [comment]
Duplicate of https://github.com/podverse/podverse-rn/issues/489

**Comment by x10an14** [comment]
I would also like to add my voice to support the prioritization of this feature =)

**Comment by mitchdowney** [comment]
@x10an14 I agree it would be a useful feature. I haven't been able to figure out how we'd do it yet...I'm not sure how to code the logic for getting the next (or previous) episode 🤔

**Comment by Trolladactyl** [comment]
Hi, Turd here I hope I am not out of line in commenting.   Would it be possible to just put a toggle that gives the user the option on how to group their list by either 1.) Podcast, ascending order or 2.) Ascending order regardless of podcast?

**Comment by mitchdowney** [comment]
@Trolladactyl we could definitely add some sort of configuration options. I haven't really thought through how this feature would be added yet though. There are a few moving parts to think about between the player, and figuring out what the next episode that should play should be, and syncing with server...realistically I don't see myself working on this until early 2023.

**Comment by mitchdowney** [comment]
We are archiving all Github issues and PRs that pertain to the previous generation Podverse. The Podverse dev team is currently 100% focused on the next generation app, and will be only working on next gen Github issues going forward.

If you would like to see a work-in-progress demo of the upcoming Podverse website, please visit https://alpha.podverse.fm. Our tentative release date is Spring 2026.

If you're a developer who may be interested in contributing to Podverse Next-Gen, please see our [Local Dev Setup documentation](https://github.com/podverse/podverse-ops/blob/v5-develop/dev/local-dev-setup.md). We can also be reached through [Matrix or Discord](https://podverse.fm/contact).

I'm sorry for the abrupt announcement, but thank you for your bug reports and feature requests.
